### DISCUSSION TITLE
Project 3

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#181] [Q&A]
Hello, I am receiving an error in my code and I don't understand why,

![image](https://user-images.githubusercontent.com/116235922/212474100-05225335-ff47-4041-9600-10d09ca19288.png)

![image](https://user-images.githubusercontent.com/116235922/212474084-3a19f43a-826e-4b8d-8c90-b2ea80ce1bae.png)

![image](https://user-images.githubusercontent.com/116235922/212474121-0ec8f2f2-3b5a-4bee-8c30-3f486c41d266.png)

**Comment by Awasthi-Adi** [comment]
Try checking your value for mf and m0, because it looks like you are trying to create an array from 0 to 0 with 500 steps; when you define t in your f_dm function.
